x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/cache/CacheFile.java

```diff
@@ -249,41 +248,36 @@ private void ensureOpen() {
     }
 
     CompletableFuture<Integer> fetchRange(
-        long position,
+        Tuple<Long, Long> range,
```

Comment:
A `Tuple` argument is a bit strange, can we pass in the two `long`s directly instead?

Reply:
Sure. I pushed 4dc76b83c10f3881a7783896dd3610bc7baae522